src/components/heading/heading.scss

```diff
@@ -66,3 +66,50 @@
   text-align: left;
   width: 50px;
 }
+
+// When inside a dialog full screen
+.carbon-dialog-full-screen__content {
+  > .carbon-carousel {
+    .carbon-heading {
+      padding-bottom: 0;
+    }
+  }
+
+  .carbon-heading {
+    height: 98px;
+
+    .carbon-heading__header {
```

Comment:
We can reduce the specificity of this `.carbon-heading_header` rule by placing it alongside `.carbon-heading` (instead of nesting it within it):

```
.carbon-dialog-full-screen__content {
  ...
  .carbon-heading {
    ...
  }
  .carbon-heading_header {
    ...
  }
}
```